apps/web/src/actions/invitations/accept.ts

```diff
@@ -43,5 +42,5 @@ export const acceptInvitationAction = createServerAction()
       },
     })
 
-    return redirect(ROUTES.root)
+    return frontendRedirect(ROUTES.root)
```

Comment:
One trade-off of `next-safe-action` that we didn't have with `zsa` is the backend next.js redirect doesn't work. I did a little abstration to overcome this but in the end redirect happens now on the frontend `onSuccess`